lib/brainstem/concerns/presenter_dsl.rb

```diff
@@ -100,6 +158,8 @@ def reset_configuration!
           configuration.nest!(:filters)
           configuration.nest!(:sort_orders)
           configuration.nest!(:associations)
```

Comment:
I think it should be for consistency. If you can you think of any reason why it would be good to have this remain heritable, I'll change it back, but until then I'll add it.